app/models/loader/orchestrate.rb

```diff
@@ -118,6 +118,67 @@ def load
       store_passwords
 
       store_public_keys
+
+      emit_audit
+    end
+
+    SDID = Audit::SDID
+
+    OPS_MAP = {
+      "INSERT" => "add",
+      "DELETE" => "remove",
+      "UPDATE" => "change"
+    }
+
+    SUBJECT_MAP = {
```

Comment:
Indicate what this is for? It doesn't appear to be used in this file.

Reply:
Yup, leftover, thanks. I changed it to the `case` below.